src/components/rangeUsePlanPage/versionsList/VersionsDropdownList.js

```diff
@@ -0,0 +1,86 @@
+import React, { useState } from 'react'
+import Box from '@material-ui/core/Box'
+import Collapse from '@material-ui/core/Collapse'
+import Table from '@material-ui/core/Table'
+import TableBody from '@material-ui/core/TableBody'
+import TableCell from '@material-ui/core/TableCell'
+import TableHead from '@material-ui/core/TableHead'
+import TableRow from '@material-ui/core/TableRow'
+import Typography from '@material-ui/core/Typography'
+import moment from 'moment'
+import classnames from 'classnames'
+import Status from '../../common/Status'
+
+const VersionsDropDownList = ({ versions, open }) => {
+  const versionOptions = versions.map(v => ({
+    key: v.version,
+    value: v,
+    text: `v${v.version}`,
+    content: (
+      <div
+        style={{
+          display: 'grid',
+          gridTemplateColumns: '40px repeat(2, auto)',
+          gridTemplateRows: '1fr'
+        }}>
+        <span>v{v.version}</span>
+        <span>{v.status.name}</span>
+        <span>{moment(v.createdAt).format('MMM DD, YYYY, h:mm:ss a')}</span>
+      </div>
+    ),
+    version: v
+  }))
+
+  return (
+    <TableRow>
+      <TableCell style={{ paddingBottom: 0, paddingTop: 0 }} colSpan={7}>
+        <Collapse in={open} timeout="auto" unmountOnExit>
+          <Box margin={0} style={{ marginBottom: '10px' }}>
+            <Typography variant="h6" gutterBottom component="div">
+              Versions
+            </Typography>
+            <Table size="small" aria-label="purchases">
+              <TableHead>
+                <TableRow>
+                  <TableCell style={{ color: 'grey' }}>Legal Start</TableCell>
+                  <TableCell style={{ color: 'grey' }}>Legal End</TableCell>
+                  <TableCell style={{ color: 'grey', align: 'left' }}>
+                    Status
+                  </TableCell>
+                </TableRow>
+              </TableHead>
+              <TableBody>
+                {versionOptions.map((option, index) => (
+                  <TableRow key={index} hover={true}>
+                    <TableCell component="th" scope="row">
+                      {moment(option.version.effectiveLegalStart).format(
+                        'MMM DD YYYY h:mm a'
+                      )}
+                    </TableCell>
+                    <TableCell>
+                      {option.version.effectiveLegalEnd == null
```

Comment:
if you rebase ontop of dev you will see this change from looking for a null end date to using the new iscurrentlegalversion var, otherwise this looks great and props to @calebissharp for a killer PR review.

Reply:
after a request from @calebissharp , i've changed it to be snake case in the db which should make it camel case for you:  `isCurrentLegalVersion`

migration will run in next 20 min or so and you'll see the change in Test.